app/mailers/output_document_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class OutputDocumentMailer < ActionMailer::Base
+  def issue(appointment_summary, output_document)
```

Comment:
It's a tough call at the moment seeing as the body of the sent email is currently blank. Additional attributes may or may not be required from the `AppointmentSummary` further down the line